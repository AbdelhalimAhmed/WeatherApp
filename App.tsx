import { StatusBar } from 'expo-status-bar';
import { NavigationContainer } from '@react-navigation/native';
import { QueryClient, QueryClientProvider } from '@tanstack/react-query';

import RootNavigator from './src/navigation/RootNavigator';

const queryClient = new QueryClient();

export default function App() {
  return (
    <NavigationContainer>
      <StatusBar style="auto" />
      <QueryClientProvider client={queryClient}>
        <RootNavigator />
      </QueryClientProvider>
    </NavigationContainer>
  );
}
