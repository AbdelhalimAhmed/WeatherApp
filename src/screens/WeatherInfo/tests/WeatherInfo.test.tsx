import React from 'react';
import { render, screen } from '@testing-library/react-native';

import { ForecastHourlyTable } from '../index';
import { MockUseCase1WhenHasTwoDaysFormattedData, MockUseCase2WhenHasTwoDaysFormattedData, MockUseCase1WhenNoEnoughDataFound } from './MocksData';
import App from '../../../../App';

describe('OpeningHours component', () => {
  it('renders correctly when data not fetched', () => {
    render(
      <App />
    );
    expect(screen.getByText("Search by city name to get weather info")).toBeTruthy();
  });
});

describe('ForecastHourlyTable component', () => {
  it("renders correctly when providing MockUseCase1WhenHasTwoDaysFormattedData which contains next 5 hours when there's only one day's data", () => {
    const forecastHourlyTable = render(<ForecastHourlyTable forecastHourly={MockUseCase1WhenHasTwoDaysFormattedData} />).toJSON();
    expect(screen.getByText("Now")).toBeTruthy();
    expect(forecastHourlyTable).toMatchSnapshot();
  });

  it("renders correctly when providing MockUseCase2WhenHasTwoDaysFormattedData which contains next 5 hours when there are two days' data", () => {
    const forecastHourlyTable = render(<ForecastHourlyTable forecastHourly={MockUseCase2WhenHasTwoDaysFormattedData} />).toJSON();
    expect(screen.getByText("Now")).toBeTruthy();
    expect(forecastHourlyTable).toMatchSnapshot();
  });

  it("renders correctly when providing MockUseCase1WhenNoEnoughDataFound which contains available data if there isn't enough data", () => {
    const forecastHourlyTable = render(<ForecastHourlyTable forecastHourly={MockUseCase1WhenNoEnoughDataFound} />).toJSON();
    expect(screen.getByText("Now")).toBeTruthy();
    expect(forecastHourlyTable).toMatchSnapshot();
  });

  it("renders correctly when providing empty array", () => {
    const forecastHourlyTable = render(<ForecastHourlyTable forecastHourly={[]} />).toJSON();
    expect(forecastHourlyTable).toMatchSnapshot();
  });
});

