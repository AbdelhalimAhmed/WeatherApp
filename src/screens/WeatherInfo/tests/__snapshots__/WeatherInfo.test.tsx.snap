// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ForecastHourlyTable component renders correctly when providing MockUseCase1WhenHasTwoDaysFormattedData which contains next 5 hours when there's only one day's data 1`] = `
<RCTScrollView
  style={
    {
      "maxHeight": 500,
    }
  }
>
  <View>
    <View
      style={
        {
          "alignItems": "center",
          "borderBottomColor": "rgb(0, 122, 255)",
          "borderBottomWidth": 0.5,
          "flex": 1,
          "flexDirection": "row",
          "gap": 16,
          "justifyContent": "space-between",
          "paddingVertical": 12,
        }
      }
    >
      <View
        style={
          {
            "flex": 1,
            "justifyContent": "center",
          }
        }
      >
        <Text
          style={
            [
              {
                "color": "#000",
                "fontSize": 16,
                "fontWeight": "bold",
                "lineHeight": 22,
                "textAlign": "left",
                "textTransform": "none",
              },
              undefined,
            ]
          }
        >
          Now
        </Text>
      </View>
      <Text
        style={
          [
            {
              "color": "rgb(0, 122, 255)",
              "fontSize": 12,
              "fontWeight": "bold",
              "lineHeight": 16,
              "textAlign": "left",
              "textTransform": "none",
            },
            {
              "textTransform": "capitalize",
            },
          ]
        }
      >
        3
        째C
      </Text>
      <Image
        source={
          {
            "uri": "https://cdn.weatherapi.com/weather/64x64/night/266.png",
          }
        }
        style={
          {
            "height": 30,
            "width": 30,
          }
        }
      />
      <View
        style={
          {
            "flex": 1,
            "justifyContent": "center",
          }
        }
      >
        <Text
          style={
            [
              {
                "color": "rgb(0, 122, 255)",
                "fontSize": 12,
                "fontWeight": "bold",
                "lineHeight": 16,
                "textAlign": "right",
                "textTransform": "uppercase",
              },
              {
                "textTransform": "capitalize",
              },
            ]
          }
        >
          Light drizzle
        </Text>
      </View>
    </View>
    <View
      style={
        {
          "alignItems": "center",
          "borderBottomColor": "rgb(0, 122, 255)",
          "borderBottomWidth": 0.5,
          "flex": 1,
          "flexDirection": "row",
          "gap": 16,
          "justifyContent": "space-between",
          "paddingVertical": 12,
        }
      }
    >
      <View
        style={
          {
            "flex": 1,
            "justifyContent": "center",
          }
        }
      >
        <Text
          style={
            [
              {
                "color": "#000",
                "fontSize": 16,
                "fontWeight": "bold",
                "lineHeight": 22,
                "textAlign": "left",
                "textTransform": "none",
              },
              undefined,
            ]
          }
        >
          01:00 AM
        </Text>
      </View>
      <Text
        style={
          [
            {
              "color": "rgb(0, 122, 255)",
              "fontSize": 12,
              "fontWeight": "bold",
              "lineHeight": 16,
              "textAlign": "left",
              "textTransform": "none",
            },
            {
              "textTransform": "capitalize",
            },
          ]
        }
      >
        1
        째C
      </Text>
      <Image
        source={
          {
            "uri": "https://cdn.weatherapi.com/weather/64x64/night/266.png",
          }
        }
        style={
          {
            "height": 30,
            "width": 30,
          }
        }
      />
      <View
        style={
          {
            "flex": 1,
            "justifyContent": "center",
          }
        }
      >
        <Text
          style={
            [
              {
                "color": "rgb(0, 122, 255)",
                "fontSize": 12,
                "fontWeight": "bold",
                "lineHeight": 16,
                "textAlign": "right",
                "textTransform": "uppercase",
              },
              {
                "textTransform": "capitalize",
              },
            ]
          }
        >
          Light drizzle
        </Text>
      </View>
    </View>
    <View
      style={
        {
          "alignItems": "center",
          "borderBottomColor": "rgb(0, 122, 255)",
          "borderBottomWidth": 0.5,
          "flex": 1,
          "flexDirection": "row",
          "gap": 16,
          "justifyContent": "space-between",
          "paddingVertical": 12,
        }
      }
    >
      <View
        style={
          {
            "flex": 1,
            "justifyContent": "center",
          }
        }
      >
        <Text
          style={
            [
              {
                "color": "#000",
                "fontSize": 16,
                "fontWeight": "bold",
                "lineHeight": 22,
                "textAlign": "left",
                "textTransform": "none",
              },
              undefined,
            ]
          }
        >
          02:00 AM
        </Text>
      </View>
      <Text
        style={
          [
            {
              "color": "rgb(0, 122, 255)",
              "fontSize": 12,
              "fontWeight": "bold",
              "lineHeight": 16,
              "textAlign": "left",
              "textTransform": "none",
            },
            {
              "textTransform": "capitalize",
            },
          ]
        }
      >
        -3
        째C
      </Text>
      <Image
        source={
          {
            "uri": "https://cdn.weatherapi.com/weather/64x64/night/266.png",
          }
        }
        style={
          {
            "height": 30,
            "width": 30,
          }
        }
      />
      <View
        style={
          {
            "flex": 1,
            "justifyContent": "center",
          }
        }
      >
        <Text
          style={
            [
              {
                "color": "rgb(0, 122, 255)",
                "fontSize": 12,
                "fontWeight": "bold",
                "lineHeight": 16,
                "textAlign": "right",
                "textTransform": "uppercase",
              },
              {
                "textTransform": "capitalize",
              },
            ]
          }
        >
          Light drizzle
        </Text>
      </View>
    </View>
    <View
      style={
        {
          "alignItems": "center",
          "borderBottomColor": "rgb(0, 122, 255)",
          "borderBottomWidth": 0.5,
          "flex": 1,
          "flexDirection": "row",
          "gap": 16,
          "justifyContent": "space-between",
          "paddingVertical": 12,
        }
      }
    >
      <View
        style={
          {
            "flex": 1,
            "justifyContent": "center",
          }
        }
      >
        <Text
          style={
            [
              {
                "color": "#000",
                "fontSize": 16,
                "fontWeight": "bold",
                "lineHeight": 22,
                "textAlign": "left",
                "textTransform": "none",
              },
              undefined,
            ]
          }
        >
          03:00 AM
        </Text>
      </View>
      <Text
        style={
          [
            {
              "color": "rgb(0, 122, 255)",
              "fontSize": 12,
              "fontWeight": "bold",
              "lineHeight": 16,
              "textAlign": "left",
              "textTransform": "none",
            },
            {
              "textTransform": "capitalize",
            },
          ]
        }
      >
        -34
        째C
      </Text>
      <Image
        source={
          {
            "uri": "https://cdn.weatherapi.com/weather/64x64/night/266.png",
          }
        }
        style={
          {
            "height": 30,
            "width": 30,
          }
        }
      />
      <View
        style={
          {
            "flex": 1,
            "justifyContent": "center",
          }
        }
      >
        <Text
          style={
            [
              {
                "color": "rgb(0, 122, 255)",
                "fontSize": 12,
                "fontWeight": "bold",
                "lineHeight": 16,
                "textAlign": "right",
                "textTransform": "uppercase",
              },
              {
                "textTransform": "capitalize",
              },
            ]
          }
        >
          Light drizzle
        </Text>
      </View>
    </View>
    <View
      style={
        {
          "alignItems": "center",
          "borderBottomColor": "rgb(0, 122, 255)",
          "borderBottomWidth": 0.5,
          "flex": 1,
          "flexDirection": "row",
          "gap": 16,
          "justifyContent": "space-between",
          "paddingVertical": 12,
        }
      }
    >
      <View
        style={
          {
            "flex": 1,
            "justifyContent": "center",
          }
        }
      >
        <Text
          style={
            [
              {
                "color": "#000",
                "fontSize": 16,
                "fontWeight": "bold",
                "lineHeight": 22,
                "textAlign": "left",
                "textTransform": "none",
              },
              undefined,
            ]
          }
        >
          04:00 AM
        </Text>
      </View>
      <Text
        style={
          [
            {
              "color": "rgb(0, 122, 255)",
              "fontSize": 12,
              "fontWeight": "bold",
              "lineHeight": 16,
              "textAlign": "left",
              "textTransform": "none",
            },
            {
              "textTransform": "capitalize",
            },
          ]
        }
      >
        34
        째C
      </Text>
      <Image
        source={
          {
            "uri": "https://cdn.weatherapi.com/weather/64x64/night/266.png",
          }
        }
        style={
          {
            "height": 30,
            "width": 30,
          }
        }
      />
      <View
        style={
          {
            "flex": 1,
            "justifyContent": "center",
          }
        }
      >
        <Text
          style={
            [
              {
                "color": "rgb(0, 122, 255)",
                "fontSize": 12,
                "fontWeight": "bold",
                "lineHeight": 16,
                "textAlign": "right",
                "textTransform": "uppercase",
              },
              {
                "textTransform": "capitalize",
              },
            ]
          }
        >
          Light drizzle
        </Text>
      </View>
    </View>
  </View>
</RCTScrollView>
`;

exports[`ForecastHourlyTable component renders correctly when providing MockUseCase1WhenNoEnoughDataFound which contains available data if there isn't enough data 1`] = `
<RCTScrollView
  style={
    {
      "maxHeight": 500,
    }
  }
>
  <View>
    <View
      style={
        {
          "alignItems": "center",
          "borderBottomColor": "rgb(0, 122, 255)",
          "borderBottomWidth": 0.5,
          "flex": 1,
          "flexDirection": "row",
          "gap": 16,
          "justifyContent": "space-between",
          "paddingVertical": 12,
        }
      }
    >
      <View
        style={
          {
            "flex": 1,
            "justifyContent": "center",
          }
        }
      >
        <Text
          style={
            [
              {
                "color": "#000",
                "fontSize": 16,
                "fontWeight": "bold",
                "lineHeight": 22,
                "textAlign": "left",
                "textTransform": "none",
              },
              undefined,
            ]
          }
        >
          Now
        </Text>
      </View>
      <Text
        style={
          [
            {
              "color": "rgb(0, 122, 255)",
              "fontSize": 12,
              "fontWeight": "bold",
              "lineHeight": 16,
              "textAlign": "left",
              "textTransform": "none",
            },
            {
              "textTransform": "capitalize",
            },
          ]
        }
      >
        89
        째C
      </Text>
      <Image
        source={
          {
            "uri": "https://cdn.weatherapi.com/weather/64x64/night/266.png",
          }
        }
        style={
          {
            "height": 30,
            "width": 30,
          }
        }
      />
      <View
        style={
          {
            "flex": 1,
            "justifyContent": "center",
          }
        }
      >
        <Text
          style={
            [
              {
                "color": "rgb(0, 122, 255)",
                "fontSize": 12,
                "fontWeight": "bold",
                "lineHeight": 16,
                "textAlign": "right",
                "textTransform": "uppercase",
              },
              {
                "textTransform": "capitalize",
              },
            ]
          }
        >
          cloudy
        </Text>
      </View>
    </View>
    <View
      style={
        {
          "alignItems": "center",
          "borderBottomColor": "rgb(0, 122, 255)",
          "borderBottomWidth": 0.5,
          "flex": 1,
          "flexDirection": "row",
          "gap": 16,
          "justifyContent": "space-between",
          "paddingVertical": 12,
        }
      }
    >
      <View
        style={
          {
            "flex": 1,
            "justifyContent": "center",
          }
        }
      >
        <Text
          style={
            [
              {
                "color": "#000",
                "fontSize": 16,
                "fontWeight": "bold",
                "lineHeight": 22,
                "textAlign": "left",
                "textTransform": "none",
              },
              undefined,
            ]
          }
        >
          07:00 AM
        </Text>
      </View>
      <Text
        style={
          [
            {
              "color": "rgb(0, 122, 255)",
              "fontSize": 12,
              "fontWeight": "bold",
              "lineHeight": 16,
              "textAlign": "left",
              "textTransform": "none",
            },
            {
              "textTransform": "capitalize",
            },
          ]
        }
      >
        -9
        째C
      </Text>
      <Image
        source={
          {
            "uri": "https://cdn.weatherapi.com/weather/64x64/night/266.png",
          }
        }
        style={
          {
            "height": 30,
            "width": 30,
          }
        }
      />
      <View
        style={
          {
            "flex": 1,
            "justifyContent": "center",
          }
        }
      >
        <Text
          style={
            [
              {
                "color": "rgb(0, 122, 255)",
                "fontSize": 12,
                "fontWeight": "bold",
                "lineHeight": 16,
                "textAlign": "right",
                "textTransform": "uppercase",
              },
              {
                "textTransform": "capitalize",
              },
            ]
          }
        >
          snow
        </Text>
      </View>
    </View>
  </View>
</RCTScrollView>
`;

exports[`ForecastHourlyTable component renders correctly when providing MockUseCase2WhenHasTwoDaysFormattedData which contains next 5 hours when there are two days' data 1`] = `
<RCTScrollView
  style={
    {
      "maxHeight": 500,
    }
  }
>
  <View>
    <View
      style={
        {
          "alignItems": "center",
          "borderBottomColor": "rgb(0, 122, 255)",
          "borderBottomWidth": 0.5,
          "flex": 1,
          "flexDirection": "row",
          "gap": 16,
          "justifyContent": "space-between",
          "paddingVertical": 12,
        }
      }
    >
      <View
        style={
          {
            "flex": 1,
            "justifyContent": "center",
          }
        }
      >
        <Text
          style={
            [
              {
                "color": "#000",
                "fontSize": 16,
                "fontWeight": "bold",
                "lineHeight": 22,
                "textAlign": "left",
                "textTransform": "none",
              },
              undefined,
            ]
          }
        >
          Now
        </Text>
      </View>
      <Text
        style={
          [
            {
              "color": "rgb(0, 122, 255)",
              "fontSize": 12,
              "fontWeight": "bold",
              "lineHeight": 16,
              "textAlign": "left",
              "textTransform": "none",
            },
            {
              "textTransform": "capitalize",
            },
          ]
        }
      >
        -9
        째C
      </Text>
      <Image
        source={
          {
            "uri": "https://cdn.weatherapi.com/weather/64x64/night/266.png",
          }
        }
        style={
          {
            "height": 30,
            "width": 30,
          }
        }
      />
      <View
        style={
          {
            "flex": 1,
            "justifyContent": "center",
          }
        }
      >
        <Text
          style={
            [
              {
                "color": "rgb(0, 122, 255)",
                "fontSize": 12,
                "fontWeight": "bold",
                "lineHeight": 16,
                "textAlign": "right",
                "textTransform": "uppercase",
              },
              {
                "textTransform": "capitalize",
              },
            ]
          }
        >
          rain
        </Text>
      </View>
    </View>
    <View
      style={
        {
          "alignItems": "center",
          "borderBottomColor": "rgb(0, 122, 255)",
          "borderBottomWidth": 0.5,
          "flex": 1,
          "flexDirection": "row",
          "gap": 16,
          "justifyContent": "space-between",
          "paddingVertical": 12,
        }
      }
    >
      <View
        style={
          {
            "flex": 1,
            "justifyContent": "center",
          }
        }
      >
        <Text
          style={
            [
              {
                "color": "#000",
                "fontSize": 16,
                "fontWeight": "bold",
                "lineHeight": 22,
                "textAlign": "left",
                "textTransform": "none",
              },
              undefined,
            ]
          }
        >
          11:00 PM
        </Text>
      </View>
      <Text
        style={
          [
            {
              "color": "rgb(0, 122, 255)",
              "fontSize": 12,
              "fontWeight": "bold",
              "lineHeight": 16,
              "textAlign": "left",
              "textTransform": "none",
            },
            {
              "textTransform": "capitalize",
            },
          ]
        }
      >
        -29
        째C
      </Text>
      <Image
        source={
          {
            "uri": "https://cdn.weatherapi.com/weather/64x64/night/266.png",
          }
        }
        style={
          {
            "height": 30,
            "width": 30,
          }
        }
      />
      <View
        style={
          {
            "flex": 1,
            "justifyContent": "center",
          }
        }
      >
        <Text
          style={
            [
              {
                "color": "rgb(0, 122, 255)",
                "fontSize": 12,
                "fontWeight": "bold",
                "lineHeight": 16,
                "textAlign": "right",
                "textTransform": "uppercase",
              },
              {
                "textTransform": "capitalize",
              },
            ]
          }
        >
          snow
        </Text>
      </View>
    </View>
    <View
      style={
        {
          "alignItems": "center",
          "borderBottomColor": "rgb(0, 122, 255)",
          "borderBottomWidth": 0.5,
          "flex": 1,
          "flexDirection": "row",
          "gap": 16,
          "justifyContent": "space-between",
          "paddingVertical": 12,
        }
      }
    >
      <View
        style={
          {
            "flex": 1,
            "justifyContent": "center",
          }
        }
      >
        <Text
          style={
            [
              {
                "color": "#000",
                "fontSize": 16,
                "fontWeight": "bold",
                "lineHeight": 22,
                "textAlign": "left",
                "textTransform": "none",
              },
              undefined,
            ]
          }
        >
          12:00 AM
        </Text>
      </View>
      <Text
        style={
          [
            {
              "color": "rgb(0, 122, 255)",
              "fontSize": 12,
              "fontWeight": "bold",
              "lineHeight": 16,
              "textAlign": "left",
              "textTransform": "none",
            },
            {
              "textTransform": "capitalize",
            },
          ]
        }
      >
        3
        째C
      </Text>
      <Image
        source={
          {
            "uri": "https://cdn.weatherapi.com/weather/64x64/night/266.png",
          }
        }
        style={
          {
            "height": 30,
            "width": 30,
          }
        }
      />
      <View
        style={
          {
            "flex": 1,
            "justifyContent": "center",
          }
        }
      >
        <Text
          style={
            [
              {
                "color": "rgb(0, 122, 255)",
                "fontSize": 12,
                "fontWeight": "bold",
                "lineHeight": 16,
                "textAlign": "right",
                "textTransform": "uppercase",
              },
              {
                "textTransform": "capitalize",
              },
            ]
          }
        >
          Light drizzle
        </Text>
      </View>
    </View>
    <View
      style={
        {
          "alignItems": "center",
          "borderBottomColor": "rgb(0, 122, 255)",
          "borderBottomWidth": 0.5,
          "flex": 1,
          "flexDirection": "row",
          "gap": 16,
          "justifyContent": "space-between",
          "paddingVertical": 12,
        }
      }
    >
      <View
        style={
          {
            "flex": 1,
            "justifyContent": "center",
          }
        }
      >
        <Text
          style={
            [
              {
                "color": "#000",
                "fontSize": 16,
                "fontWeight": "bold",
                "lineHeight": 22,
                "textAlign": "left",
                "textTransform": "none",
              },
              undefined,
            ]
          }
        >
          01:00 AM
        </Text>
      </View>
      <Text
        style={
          [
            {
              "color": "rgb(0, 122, 255)",
              "fontSize": 12,
              "fontWeight": "bold",
              "lineHeight": 16,
              "textAlign": "left",
              "textTransform": "none",
            },
            {
              "textTransform": "capitalize",
            },
          ]
        }
      >
        1
        째C
      </Text>
      <Image
        source={
          {
            "uri": "https://cdn.weatherapi.com/weather/64x64/night/266.png",
          }
        }
        style={
          {
            "height": 30,
            "width": 30,
          }
        }
      />
      <View
        style={
          {
            "flex": 1,
            "justifyContent": "center",
          }
        }
      >
        <Text
          style={
            [
              {
                "color": "rgb(0, 122, 255)",
                "fontSize": 12,
                "fontWeight": "bold",
                "lineHeight": 16,
                "textAlign": "right",
                "textTransform": "uppercase",
              },
              {
                "textTransform": "capitalize",
              },
            ]
          }
        >
          Light drizzle
        </Text>
      </View>
    </View>
    <View
      style={
        {
          "alignItems": "center",
          "borderBottomColor": "rgb(0, 122, 255)",
          "borderBottomWidth": 0.5,
          "flex": 1,
          "flexDirection": "row",
          "gap": 16,
          "justifyContent": "space-between",
          "paddingVertical": 12,
        }
      }
    >
      <View
        style={
          {
            "flex": 1,
            "justifyContent": "center",
          }
        }
      >
        <Text
          style={
            [
              {
                "color": "#000",
                "fontSize": 16,
                "fontWeight": "bold",
                "lineHeight": 22,
                "textAlign": "left",
                "textTransform": "none",
              },
              undefined,
            ]
          }
        >
          02:00 AM
        </Text>
      </View>
      <Text
        style={
          [
            {
              "color": "rgb(0, 122, 255)",
              "fontSize": 12,
              "fontWeight": "bold",
              "lineHeight": 16,
              "textAlign": "left",
              "textTransform": "none",
            },
            {
              "textTransform": "capitalize",
            },
          ]
        }
      >
        -3
        째C
      </Text>
      <Image
        source={
          {
            "uri": "https://cdn.weatherapi.com/weather/64x64/night/266.png",
          }
        }
        style={
          {
            "height": 30,
            "width": 30,
          }
        }
      />
      <View
        style={
          {
            "flex": 1,
            "justifyContent": "center",
          }
        }
      >
        <Text
          style={
            [
              {
                "color": "rgb(0, 122, 255)",
                "fontSize": 12,
                "fontWeight": "bold",
                "lineHeight": 16,
                "textAlign": "right",
                "textTransform": "uppercase",
              },
              {
                "textTransform": "capitalize",
              },
            ]
          }
        >
          Light drizzle
        </Text>
      </View>
    </View>
  </View>
</RCTScrollView>
`;

exports[`ForecastHourlyTable component renders correctly when providing empty array 1`] = `
<RCTScrollView
  style={
    {
      "maxHeight": 500,
    }
  }
>
  <View />
</RCTScrollView>
`;
